src/main/scala/com/databricks/spark/csv/DefaultSource.scala

```diff
@@ -110,6 +110,14 @@ class DefaultSource
     } else {
       throw new Exception("Ignore white space flag can be true or false")
     }
+    val treatEmptyValuesAsNulls = parameters.getOrElse("treatEmptyValuesAsNulls", "false")
+    val treatEmptyValuesAsNullsFlag = if(treatEmptyValuesAsNulls == "false") {
```

Comment:
Style nit: ```if (treatEmptyValuesAsNulls ...```
Leave a space between the parenthesis and if here and everywhere else in the PR.